google/cloud/forseti/services/dao.py

```diff
@@ -608,10 +609,15 @@ def expand_special_members(cls, session):
                     session, '', member_types=member_type_map.keys()):
                 member_type, project_id = parent_member.split('/')
                 role = member_type_map[member_type]
-                iam_policy = cls.get_iam_policy(
-                    session,
-                    'project/{}'.format(project_id),
-                    roles=[role])
+                try:
```

Comment:
You can add a test for this by adding a member to an iam binding in /tests/services/inventory/test_data/additional_cai_iam_policies.dump to bq_test_ds that points to a non-existent project.

'projectEditor:project_does_not_exist'

Then you'll need to regenerate the test files like this from a virtualenv that has the forseti dependencies installed:

PYTHONPATH=. python tests/services/inventory/update_cai_dumps.py
PYTHONPATH=. python tests/services/model/importer/update_test_dbs.py

Then the tests should still pass, particularly /tests/services/model/importer/importer_test.py.

To further test, update https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/tests/services/test_models.py#L57 to add a binding for projecteditor/project_does_not_exist to one of the buckets.

Then all tests in /tests/services/dao_test.py should still pass.